health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/UserDownload.js

```
@@ -0,0 +1,100 @@
+                            title=""
+                            fileName={`FileNo${item?.fileStoreId.slice(0, 4)}`}
+                            downloadHandler={() => {
+                                Digit.Utils.campaign.downloadExcelWithCustomName({
+                                    fileStoreId: item?.fileStoreId, customName: `FileNo${item?.fileStoreId.slice(0, 4)}`
+                                });
+                            }} // Passing the download function
+                            auditDetails={{userName:item?.username,lastmodTime:item?.auditDetails?.lastmodtime}}
+                        />
+                        )
+
+})
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)